pkg/operator/controller/certificate/publish_ca.go

```diff
@@ -52,10 +56,32 @@ func (r *reconciler) ensureRouterCAConfigMap(secret *corev1.Secret, ingresses []
 	return nil
 }
 
-// desiredRouterCAConfigMap returns the desired router CA configmap.
-func desiredRouterCAConfigMap(secret *corev1.Secret, ingresses []operatorv1.IngressController) (*corev1.ConfigMap, error) {
-	if !shouldPublishRouterCA(ingresses) {
-		return nil, nil
+// desiredRouterCAConfigMap returns a configmap containing a single 'ca-bundle.crt'
+// key whose value is the concatenation of:
+//
+// 1. The default generated CA tls.crt data
+// 2. All distinct default certificate secret tls.crt data referenced by ingresscontrollers
+func (r *reconciler) desiredRouterCAConfigMap(ingresses []operatorv1.IngressController) (*corev1.ConfigMap, error) {
+	caName := controller.RouterCASecretName(r.operatorNamespace)
+	secrets := map[string]types.NamespacedName{
+		caName.String(): caName,
+	}
+	for _, ingress := range ingresses {
+		name := controller.RouterEffectiveDefaultCertificateSecretName(&ingress)
```

Comment:
@Miciah @enj When, if ever, is it appropriate to publish the default generated certificate?

Reply:
In this PR's current implementation, the default generated certificate data is published when it's referenced by an ingresscontroller just like any other reference.

Reply:
> When, if ever, is it appropriate to publish the default generated certificate?

Never.  No consumer of `router-ca` should need the default certificate.  (It is in `router-certs` for consumers who do need it.)

Reply:
> the default generated certificate data is published when it's referenced by an ingresscontroller

It should only exist when it's referenced by an ingresscontroller, and it should be published in `router-certs` but not in `router-ca`.

Reply:
Fixed.